src/logics/ArithLogic.h

```diff
@@ -346,7 +355,7 @@ class ArithLogic : public Logic {
     bool isLinearTerm(PTRef tr) const;
     bool isLinearFactor(PTRef tr) const;
     pair<Number, vec<PTRef>> getConstantAndFactors(PTRef sum) const;
-    pair<PTRef, PTRef> splitTermToVarAndConst(PTRef term) const;
+    pair<PTRef, PTRef> splitPolyTerm(PTRef term) const;
```

Comment:
Can you add a docstring here to document that it splits the term into a monomial and its coefficient?